package.json

```diff
@@ -21,12 +22,17 @@
     "@types/lodash": "^4.14.47",
     "@types/mocha": "^2.2.36",
     "@types/node": "^7.0.0",
+    "@types/yargs": "^6.5.0",
     "chai": "^3.5.0",
     "mocha": "^3.2.0",
-    "ts-node": "^2.0.0"
+    "ts-node": "^2.0.0",
+    "typescript": "^2.1.4"
```

Comment:
Intention: ensuring users would not be locked in to a specific TS version (peer dependency). Evidently it would still be needed at run-time though. This was my idea of addressing that.
I'm open to ideas, just hope we wouldn't need to force every single DefinitelyTyped repo to force adhering one and the same TypeScript version.
For Ramda, for one, I'm highly dependent on fixes to the outstanding problems in TypeScript. I'd actually consider trying out different branches / forks for that.